cupy/core/raw.pyx

```diff
@@ -21,17 +21,21 @@ cdef class RawKernel:
         name (str): Name of the kernel function.
         options (str): Compile options passed to NVRTC. For details, see
             https://docs.nvidia.com/cuda/nvrtc/index.html#group__options.
+        backend (str): {"nvrtc", "nvcc"}. Defaults to "nvcc"
```

Comment:
The docstring should be `Defaults to "nvrtc"`, not "nvcc".

Reply:
fixed